fast_com_example_usage.py

```diff
@@ -1,8 +1,7 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
```

Comment:
I suggest adding `from __future__ import print_function` here too for full 2-3 compatibility.